core/src/main/java/org/apache/calcite/sql/fun/SqlMonotonicBinaryOperator.java

```diff
@@ -66,6 +66,10 @@ public SqlMonotonicBinaryOperator(
 
```

Comment:
Not sure, how about move the check above like below:
```
    if ((mono1 == SqlMonotonicity.CONSTANT)
        && (mono0 == SqlMonotonicity.CONSTANT)
        || call.isOperandNull(0, true) || call.isOperandNull(1, true)) {
      return SqlMonotonicity.CONSTANT;
    }
```
Such we can save the below change.

Reply:
I agree. Thanks, it is a simple way and the logic is right for current SqlMonotonicBinaryOperators("+"/"-"/"*").